internal/provider/resource_password.go

```diff
@@ -1,22 +1,109 @@
 package provider
 
 import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"golang.org/x/crypto/bcrypt"
 )
 
 func resourcePassword() *schema.Resource {
+	create := func(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
```

Comment:
Nitpick: Why is this function defined inside inside the other function (lamba)?

I can't spot anything that would require this approach ❓

Reply:
Have removed.